soknad-omsorgspenger-utbetaling/src/main/java/no/nav/k9/søknad/omsorgspenger/utbetaling/OmsorgspengerUtbetalingSøknad.java

```diff
@@ -1,64 +1,90 @@
 package no.nav.k9.søknad.omsorgspenger.utbetaling;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.*;
 import no.nav.k9.søknad.JsonUtils;
-import no.nav.k9.søknad.felles.Barn;
-import no.nav.k9.søknad.felles.Søker;
-import no.nav.k9.søknad.felles.SøknadId;
-import no.nav.k9.søknad.felles.Versjon;
+import no.nav.k9.søknad.felles.*;
 
+import javax.validation.Valid;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
+import javax.validation.constraints.NotNull;
 import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.NONE, getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE)
 public class OmsorgspengerUtbetalingSøknad {
+
+    @JsonProperty("søknadId")
+    @Valid
+    @NotNull
     public final SøknadId søknadId;
 
+    @JsonProperty("versjon")
+    @Valid
+    @NotNull
     public final Versjon versjon;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX", timezone = "UTC")
+    @JsonProperty("mottattDato")
+    @Valid
+    @NotNull
     public final ZonedDateTime mottattDato;
 
+    @JsonProperty("søker")
+    @Valid
+    @NotNull
     public final Søker søker;
 
+    @JsonProperty("barn")
```

Comment:
skal denne være notempty? eller hvorfor tar vi hensyn til barna her, men ikke på refusjonskrav?

Reply:
Den skal kunne være tom ja. I dette tilfelle så er det snakk om fosterbarn.

Reply:
Bør den ikke renames til å beskrive hva det er funksjonelt? F.eks hvis det kun er fosterbarn som skal hit så bør navnet også reflektere det?

Reply:
Egentlig bør vi bli kvitt feltet - hvis årskvantum ikke kan hente inn eller vurdere selv.  Gir ikke helt mening, når de ikke kan gjøre dette når det kommer fra arbeidsgiver.  
Fint om det renames ja

Reply:
Enig med at det bør representere det som er funksjonelt.
Dette feltet har vært laget før denne pr-en. Vet ikke hvorfor det er kalt for barn istedefor fosterbarn. Kanskje @fraadsbrandth har noe formening?